packages/create-sitecore-jss/src/templates/vue/data/content/Styleguide/ContentListField/Item1/en.yml

```diff
@@ -0,0 +1,6 @@
+id: styleguide-content-list-field-shared-1
+displayName: Styleguide Content List Item 1 (Shared)
+# Template defines the available fields. See /sitecore/definitions/templates/Styleguide-ContentList-Template.sitecore.js
+template: JssVueWeb-Styleguide-ContentList-Item-Template
```

Comment:
Need to replace `JssVueWeb-` prefix with our ejs helper in all cases `<%- helper.getAppPrefix(appPrefix, appName)%>`